hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/snapshot/SnapshotDiffReport.java

```diff
@@ -182,9 +183,16 @@ public static DiffReportEntry fromProtobuf(
    */
   private final List<DiffReportEntry> diffList;
 
-  public SnapshotDiffReport(final String volumeName, final String bucketName,
-                            final String fromSnapshot, final String toSnapshot,
-                            List<DiffReportEntry> entryList) {
+  /**
+   * subsequent token for the diff report.
+   */
+  private final String token = null; //TODO: will set it properly in HDDS-7548
```

Comment:
nitpick: TODO on a separate line on top. Let us also put "SNAPSHOT" label on all TODOs. We can scan for all pending Snapshot TODOs in our code base easily that way.